app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+
+from contextlib import contextmanager
+
+
+@contextmanager
```

Comment:
The task requires a function named `move_file`, but it should be a regular function, not a context manager. Please refactor your code to define `move_file` as a standard function.